docs/ref/templates/builtins.txt

```diff
@@ -1732,13 +1732,29 @@ example, when the active locale is ``en`` (English):
 ``34232.00`` ``{{ value|floatformat:"-3g" }}`` ``34,232``
 ============ ================================= =============
 
+Output is always localized unless the argument passed to ``floatformat`` has
```

Comment:
“... including irrespectively of the {% localize %} tag” ... ?

Reply:
Thanks, I added a note about the `{% localize off %}` tag.